File: 30 Days of code/Java/Day 13/soln.java
import java.util.*;


abstract class Book {
    String title;
    String author;
    Book(String t, String a) {
        title  = t;
        author = a;
    }
    abstract void display();
}

class MyBook extends Book {
    int price;
    
    MyBook(String title, String author, int price) {
        super(title, author);
        this.price = price;
    }
    
    void display() {
        System.out.println("Title: "  + title);
        System.out.println("Author: " + author);
        System.out.println("Price: "  + price);
    }
}
// End class
public class Solution {
   
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String title = scanner.nextLine();
        String author = scanner.nextLine();
        int price = scanner.nextInt();
        scanner.close();

        Book book = new MyBook(title, author, price);
        book.display();
    }
}

File: 30 Days of code/C/Day 13/soln.c
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <string>
using namespace std;

class Book{
    protected:
        string title;
        string author;
    public:
        Book(string t,string a){
            title=t;
            author=a;
        }
        virtual void display()=0;

};

//Write MyBook class
class MyBook : Book{
    private:
    int price;
    public:
    MyBook(string t,string a,int price):Book(t,a){
        this->price = price;
    }
    void display(){
        cout << "Title: " << title << endl << "Author: " << author << endl << "Price: " << price << endl;
    }

};
